lesson5/index.md

```diff
@@ -0,0 +1,260 @@
+---
+title: Recap Functions & Methods
+nav_order: 5
+has_children: true
+---
+
+# Lesson 4: Recap Functions & Methods
+
+## Goals
+* Recap method `signature`, `arguments` and `return` value
+* Recap `recursion`
+* Code, code, code 🤩
+
+## Recap & Homework check
+Let's look into the homework from lesson 5
+
+## Java Method
+
+in computer programming, a function is a block of code that performs a specific task.
+In object-oriented programming, the method is just another name used for function. Methods are bound to a class and they 
+define the behavior of a class. In Java, all function definitions must be inside classes.
+Let's look at an example of the main method
+
+```java
+public class Main {
+    public static void main(String[] args){
+      //code to execute
+    }
+}
+```
+
+Here, we have created a method named `main`. 
+We can see that we have used the `public`, `static` and `void` before the method name.
+
+- `public` - access modifier. It means the method can be accessed from anywhere.
+- `static` here means this method belongs to the class Main and not to a specific instance of Main. 
```

Comment:
potentially will cause confusion - but as long as we're aware and anticipating this it's fine to add I think

Reply:
Yeah I plan to mention it briefly with a tiny example. But the scope is still writing `static` methods.